packages/console/src/components/EmptyDataPlaceholder/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { AppearanceMode } from '@logto/schemas';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import EmptyDark from '@/assets/images/table-empty-dark.svg';
+import Empty from '@/assets/images/table-empty.svg';
+import { useTheme } from '@/hooks/use-theme';
+
+import * as styles from './index.module.scss';
+
+export type Props = {
+  title?: string;
+  size?: 'default' | 'small';
```

Comment:
nit: large | small? default as large

Reply:
Updated.
We have three sizes for this placeholder, the image size for the large is 256x256, the medium is 200x200, and 128x128 for the small.
Fully implemented.